_tests/travis-checks

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+# Execute Markdown Linter
+mdl README.md _posts _pages --style _linter/markdown-linter-style.rb
+
+# Github Pages needs the baseurl field to be "m-lab.github.io" to render properly in forked 
+# repos, but when we deploy to production, we need the baseurl field to be empty. This 
+# command replaces "m-lab.github.io" with an empty string.
+sed -e '/^baseurl/s/\/m-lab\.github\.io//' _config.yml > _config-deploy.yml
+
+# sets --disable-external flag for HTMLProofer if quick argument exists
```

Comment:
Prefer to just go more generic and say "skips validation checks that take a long time" because we will likely have other checks in the future that we want to run quickly in the pre-commit, but go more exhaustively in Travis